database/src/main/resources/db/migration/V1_169__change_address_post_code.sql

```diff
@@ -0,0 +1,65 @@
+ALTER TYPE address
+    ADD ATTRIBUTE "foreign_postal_code" VARCHAR(10);
+
+-- Create backup for "transport-service" table
+CREATE TABLE "transport-service_backup" AS TABLE "transport-service";
+
+CREATE OR REPLACE FUNCTION convert_transport_service_address_postalcodes()
```

Comment:
Here a sql function is created and stored to db scema.
Is that needed only once ever? If so, instead of creating and storing a function coudn't the same code be  run here using a `DO` block? Example in e.g. file `/mmtis-national-access-point/database/src/main/resources/db/migration/V1_135__detected_route_change_table.sql`

Reply:
Alternatively, if `DO`is not used should the function be dropped in the end of the migration script?

Reply:
I'll drop function afterwards